codegen/stub_api.py

```diff
@@ -730,8 +740,8 @@ def _process_module(self, node: ElementTree.Element, indent: str = "") -> str:
                     attributes += f"{indent}    {classname} ({classname}):\n"
                     attributes += f"{indent}        EnSight {classname} proxy class\n\n"
                     self._imports += self._process_class(child)
-                    classes += f"{indent}    self.{classname}: Type[{classname}] = {classname}\n"
-
+                    classes += f"{indent}    self.{classname}: '{classname}' = {classname}\n"
```

Comment:
The autocompletion doesn't like "Type". So I am using the '' syntax, which fixes the autocompletion for all the cores objects